src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -241,24 +244,26 @@ def execute(self, emulator=None):
 
         os.environ.update(envOverride)
 
-        returncode = subprocess.call(args, stdout=stdoutHandle, stderr=stderrHandle)
+        returnCode = subprocess.call(args, stdout=stdoutHandle, stderr=stderrHandle)
+        returnMessage = None
 
-        if returncode != 0:
+        if returnCode != 0:
             argsDump = {'arguments': args}
             msg = "Error running cmsRun\n%s\n" % argsDump
             try:
                 self.report.parse(jobReportXML, stepName=self.stepName)
-                returncode = self.report.getStepExitCode(stepName=self.stepName)
-                msg += "CMSSW Return code: %s\n" % returncode
+                (returnCode, returnMessage) = \
```

Comment:
Sometimes I prefer not to respect the 80 chars line length, just so the code is easier to read. This seems to be a good one for that :-)